plugin/ochttp/server.go

```diff
@@ -68,6 +72,17 @@ type Handler struct {
 	FormatSpanName func(*http.Request) string
 }
 
+// NewHandler wraps the given handler to add stats and tracing instrumentation.
+// By default, W3C TraceContext propagation format is used and the returned
+// handler is assumed to be a public endpoint.
+func NewHandler(next http.Handler) *Handler {
+	return &Handler{
+		Handler:          next,
+		Propagation:      defaultFormat,
+		IsPublicEndpoint: true,
```

Comment:
It is less likely that a server is a a public endpoint.

Reply:
I would guess that it's really a lot *more* likely to be public for most users, especially the smaller users that are not running complex microservice architectures. It is also a safer default.